cmd/envtool/main.go

```diff
@@ -216,14 +190,72 @@ func run(ctx context.Context, logger *zap.SugaredLogger) error {
 		}
 	}
 
+	return nil
+}
+
+// setupTigris configures Tigris.
+func setupTigris(ctx context.Context, logger *zap.SugaredLogger) error {
+	logger = logger.Named("tigris")
+
+	if err := waitForPort(ctx, logger, 8081); err != nil {
+		return err
+	}
+
+	cfg := &config.Driver{
+		URL: "127.0.0.1:8081",
```

Comment:
Is this port `8081` a Tigris default port (like `5432` for Postgres)? The number `8081` is quite a common number for some sample apps, and I wonder if it's a good choice to take it.

Reply:
It is, yeah:
* https://docs.tigrisdata.com/quickstart#startup-tigris-on-your-local-machine
* https://github.com/tigrisdata/tigris#building-with-docker-containers

We could map that port to some other port if there is a conflict, but let's wait for the conflict to happen